lib/task-data/tasks/node-sleep.js

```diff
@@ -0,0 +1,17 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Node sleep',
+    injectableName: 'Task.Node.Sleep',
+    implementsTask: 'Task.Base.Linux.Commands',
+    options: {
+        duration: 10, //in seconds
+        commands: ['sudo sleep {{options.duration}}']
```

Comment:
There is difference between RackHD delay and node delay. In the case a delay after reboot, RackHD delay starts from reboot finished while a node delay can only start after microkernel is ready.